torchmdnet/models/model.py

```diff
@@ -154,42 +154,64 @@ def load_model(filepath, args=None, device="cpu", **kwargs):
     if args is None:
         args = ckpt["hyper_parameters"]
 
+    delta_learning = args["remove_ref_energy"] if "remove_ref_energy" in args else False
+
     for key, value in kwargs.items():
         if not key in args:
             warnings.warn(f"Unknown hyperparameter: {key}={value}")
         args[key] = value
 
     model = create_model(args)
+    if delta_learning and "remove_ref_energy" in kwargs:
+        if not kwargs["remove_ref_energy"]:
+            assert len(model.prior_model) > 0, "Atomref prior must be added during training (with enable=False) for total energy prediction."
+            # Set the Atomref prior to enabled
+            model.prior_model[-1].enable = True
```

Comment:
This seems risky. Why do you assume the Atomref prior is the last? Maybe check the list of prior models with `isinstance(model.prior_model[x], AtomRef)` to make sure it's an AtomRef class and set them all to True?

Reply:
This is a good idea, I should check the type at least. It is technically possible to include two Atomrefs (why you would do that I do not know, though). In train.py the silent Atomref is added last always.